index.bs

```diff
@@ -285,6 +287,18 @@ Try to construct a table with the columns of the properties of <var>tabularData<
 
 Perform Logger("warn", <var>data</var>).
 
+<h4 id="dir" oldids="dir-data,dom-console-dir" method for="console">dir(<var>data</var>)</h4>
```

Comment:
What exactly are they for (or residual from) anyways? Can they be removed from the `group` section too without harming things?